java/de/yessoft/android/mohammad/ui/MainActivity/fragments/MySpecialORM.java

```diff
@@ -0,0 +1,42 @@
+package de.yessoft.android.mohammad.ui.MainActivity.fragments;
+
+import android.util.Log;
+
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+
+import java.util.List;
+
+import de.yessoft.android.mohammad.model.NewsItem;
+import de.yessoft.android.mohammad.ui.MainActivity.fragments.NewsFeedRV.NewsFeedAdapter;
+
+class MySpecialORM {
+    private static final String TAG = "ML_MySpecialORM";
```

Comment:
In general we should have the following structure  : 

model 
repository -> communicating with firebase
service  -> use the repository
all other places should use the services ,

think about the layers

Reply:
Gona do that, i need to develop my style of coding now more than ever, what is good is that i keep these things in mind